arrows/core/video_input_image_list.cxx

```diff
@@ -23,182 +23,192 @@
  * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
  * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
  * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
- * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
- * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+ * DAMAGE.
  */
 
 #include "video_input_image_list.h"
 
-#include <vital/vital_types.h>
-#include <vital/types/timestamp.h>
-#include <vital/types/image_container.h>
-#include <vital/types/image.h>
-#include <vital/types/metadata_traits.h>
 #include <vital/algo/image_io.h>
-#include <vital/exceptions.h>
+
 #include <vital/util/data_stream_reader.h>
 #include <vital/util/tokenize.h>
 
+#include <vital/types/image.h>
+#include <vital/types/image_container.h>
+#include <vital/types/metadata_traits.h>
+#include <vital/types/timestamp.h>
+
+#include <vital/exceptions.h>
+#include <vital/vital_types.h>
+
+#include <vital/range/iota.h>
+
 #include <kwiversys/Directory.hxx>
 #include <kwiversys/SystemTools.hxx>
 
 #include <algorithm>
+#include <fstream>
 #include <string>
 #include <vector>
-#include <stdint.h>
-#include <fstream>
+
+#include <cstdint>
+
+namespace kv = kwiver::vital;
+namespace kvr = kwiver::vital::range;
+
+using super = kv::algo::video_input;
```

Comment:
I'm not  a particularly big fan of `super` as a type alias.  I'd prefer not to have type aliases that are reused but defined differently in different files.  It makes it less obvious what `super` actually is.  Can we just use `kv::algo::video_input` or just `video_input`?

Reply:
If anyone ever uses a type alias `super` to refer to something *other* than the class's parent type, I will hurt them :grin:¹. But, eh, it's only used one place (I thought it was more...) and isn't needed for line length.

(¹ Ahem. The emoji I'm *really* looking for can be found [here](http://freefall.purrsia.com/ff100/fv00098.htm), in the third panel.)